docs/table_design/expression_partitioning.md

```diff
@@ -0,0 +1,256 @@
+INSERT OVERWRITE site_access1 PARTITION(event_day='2022-06-08 20:12:04')
+    SELECT * FROM site_access2 PARTITION(p20220608);
+```
+
+If you use column expression at table creation and want to overwrite data in a specific partition, you need to provide the partition column values that the partition contains. If the partition does not exist, it can be automatically created during data loading.
+
+```SQL
+INSERT OVERWRITE t_recharge_detail1 PARTITION(dt='2022-04-02',city='texas')
+    SELECT * FROM t_recharge_detail2 PARTITION(p20220402_texas);
+```
+
+### View partitions
+
+When you want to view specific information about automatically created partitions, you need to use the `SHOW PARTITIONS FROM <table_name>` statement. The `SHOW CREATE TABLE <table_name>` statement only returns the syntax for expression partitioning that is configured at table creation.
+
+```SQL
+MySQL > SHOW PARTITIONS FROM t_recharge_detail1;
++-------------+-------------------+----------------+---------------------+--------------------+--------+--------------+-----------------------------+-----------------+---------+----------------+---------------+---------------------+--------------------------+----------+------------+----------+
+| PartitionId | PartitionName     | VisibleVersion | VisibleVersionTime  | VisibleVersionHash | State  | PartitionKey | List                        | DistributionKey | Buckets | ReplicationNum | StorageMedium | CooldownTime        | LastConsistencyCheckTime | DataSize | IsInMemory | RowCount |
++-------------+-------------------+----------------+---------------------+--------------------+--------+--------------+-----------------------------+-----------------+---------+----------------+---------------+---------------------+--------------------------+----------+------------+----------+
+| 16890       | p20220401_Houston | 2              | 2023-07-19 17:24:53 | 0                  | NORMAL | dt, city     | (('2022-04-01', 'Houston')) | id              | 6       | 3              | HDD           | 9999-12-31 23:59:59 | NULL                     | 2.5KB    | false      | 1        |
+| 17056       | p20220402_texas   | 2              | 2023-07-19 17:27:42 | 0                  | NORMAL | dt, city     | (('2022-04-02', 'texas'))   | id              | 6       | 3              | HDD           | 9999-12-31 23:59:59 | NULL                     | 2.5KB    | false      | 1        |
++-------------+-------------------+----------------+---------------------+--------------------+--------+--------------+-----------------------------+-----------------+---------+----------------+---------------+---------------------+--------------------------+----------+------------+----------+
+2 rows in set (0.00 sec)
+```
+
+## Limits
+
+- Since v3.1, StarRocks's [shared-data mode](../deployment/deploy_shared_data.md) supports the time function expression and does not support the column expression.
+- Currently, using CTAS to create a table that use the expression partitioning.
```

Comment:
![image](https://github.com/StarRocks/starrocks/assets/57167462/e36ed350-361f-4582-a8e9-133ee8ab2562)
![image](https://github.com/StarRocks/starrocks/assets/57167462/c15de361-9a48-4e0d-943e-421356754587)
The English text "using CTAS to create a table that use the expression partitioning" is not a complete sentence, and it does not convey what the Chinese equivalent has stated.